sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/AzureConfiguration.java

```diff
@@ -3,13 +3,59 @@
 
 package com.azure.data.appconfiguration;
 
+import com.azure.core.util.logging.ClientLogger;
+import java.io.InputStream;
+import java.util.Properties;
+
 /**
  * Gets the SDK information for this library component.
  */
 class AzureConfiguration {
-    //TODO: Eventually remove these hardcoded strings with https://github.com/Azure/azure-sdk-for-java/issues/3141
-    static final String NAME = "application-configuration";
-    // {x-version-update-start;com.azure:azure-data-appconfiguration;current}
-    static final String VERSION = "1.0.0-preview.6";
-    // {x-version-update-end}
+
+    private static final ClientLogger LOGGER = new ClientLogger(AzureConfiguration.class);
```

Comment:
interesting. Does this trigger checkstyle  error?

Reply:
It should be non-static and use `logger` as the name

Reply:
Since it's used in a static code block, it has to be static and hence uppercase `LOGGER`.